docs/run/nodes/nodes.md

```diff
@@ -266,12 +266,20 @@ The Ceramic daemon serves an HTTP API that clients use to interact with your Cer
 
 Ceramic nodes rely on IPFS for networking. IPFS nodes connect to each other using a Libp2p module called "switch" (aka "swarm"). This module operates over a websocket, on port `4011` by default. The websocket port must be accessible to the internet so your Ceramic node can be connected to the network.
 
+!!! warning ""
+
+    IPFS nodes connect to other nodes over the p2p network, and incoming connections to a node running on a local machine might get flagged as suspicious by a router or anti-virus software. Make sure to vet all such reports and verify on a case-by-case basis whether they are false positives and can be safely ignored.
+
 !!! warning ""
 
     Healthchecks can be run against the `HEALTHCHECK_PORT` (port `8011` by default) when `HEALTHCHECK_ENABLED` is `true`.
 
 Additionally, when running IPFS the IPFS API port must be accessible by the Ceramic node. The default API port is `5001`. The IPFS node address will then be passed to Ceramic with the `ipfs.host` option in the Ceramic daemon config file.
 
+!!! warning ""
+
+    Access to the IPFS API port _must_ be restricted to the Ceramic node in order to prevent malicious API calls.
```

Comment:
```suggestion
    Access to the IPFS API port _should_ be restricted to the Ceramic node in order to prevent malicious API calls.
```

?
It kind of depends on how the person plans to use their IPFS instance. but maybe the suggestion should be more along the lines of, "you don't want to expose this to the open internet. we recommend a firewall to only allow access by your ceramic instance"

Reply:
That's fair lol. Updated.

Reply:
From the kubo docs 🤷🏼 
![image](https://user-images.githubusercontent.com/2236875/213011900-b7a65951-ab42-4eb0-b536-5999bef26357.png)